### PR TITLE
Home task: HTML 01

### PR DESCRIPTION
**Pull request by viktarradzin**

### CODE REVIEW

File: vradzin/01_HTML/task_5.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<h3 align="center">Магазин работает:</h3>
+<p align="center">c 08<sup><u>30</u></sup>до 18<sup><u>30</u></sup></br>
+обед с 13<sup><u>00</u></sup> до 14<sup><u>00</u></sup></br>
+выходной: <u>суббота</u>, <u>воскресенье</u></p>
+<hr>
+<h2>Вот такой текст</h2>
+<h2><sup>Э</sup><sub>т</sub><sup>о</sup> <sub>п</sub><sup>р</sup><sub>р</sub><sup>о</sup><sub>с</sub><sup>т</sup><sub>о</sub> <sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub> <sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></h2>
+<hr>
+<h2>Акроним и курсив</h2>
+<p>ТФ: Y(x)=e<sup>iπx</sup>­+log<sub>x</sub>4</p>
```
Comment (pluhin) [inline vradzin/01_HTML/task_5.html]: Pi char -> mnemonic

File: vradzin/01_HTML/task_5.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<hr>
+<h2>Тег small внтури абзаца</h2>
+<p align="right"><small>Создано при помощи блокнота©</small></p>
```
Comment (pluhin) [inline vradzin/01_HTML/task_5.html]: copyright  char -> mnemonic

File: vradzin/02_HTML/task3.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    </div>
+    <h2>Меню список</h2>
+    <ul>
+        <li><a href="#">Главная</a></li>
+        <li><a href="#">Каталог</a></li>
+        <li><a href="#">Акции</a></li>
+        <li><a href="#">О нас</a></li>
+      </ul>
+    <h2>Меню с вложенным списком</h2>
+    <ul>
```
Comment (pluhin) [inline vradzin/02_HTML/task3.html:23]: Wrong list structure, could you please check all these section

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vradzin/01_HTML/task_5.html (hunk 1 above)
Comment (viktarradzin) [reply vradzin/01_HTML/task_5.html]: add &pi

File: vradzin/01_HTML/task_5.html (hunk 2 above)
Comment (viktarradzin) [reply vradzin/01_HTML/task_5.html]: add &copy;

File: vradzin/02_HTML/task3.html (hunk 3 above)
Comment (viktarradzin) [reply vradzin/02_HTML/task3.html:23]: made a correction

File: vradzin/03_HTML/task_2.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    <fieldset>
+        <legend>Координаты:</legend>
+        <input type="email" name="email" id="email" placeholder="введите адрес почты" required><label for="email">email:проверка должна осуществляться при помощи новых атрибутов тега</label></br>
+        <input type="url" name="url" id="url"placeholder="введите адрес сайта" required><label for="url">url:проверка должна осуществляться при помощи новых атрибутов тега</label>
+    </fieldset>  
+    <fieldset>
+        <legend>Цифры:</legend>
+        <input type="number" id="num" name="num"   min="-2.5" max="2.5" step="0.1"><label for="num">number:Диапозон от -2.5 до 2.5 с шагом 0.1</label></br>
+        <input type="range" id="start" name="volume"  min="0" max="10"><label for="volume">range:проверка должна осуществляться при помощи новых атрибутов тега input</label>
+       min="10" max="100">
```
Comment (pluhin) [inline vradzin/03_HTML/task_2.html]: What is it?

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vradzin/03_HTML/task_2.html (hunk 4 above)
Comment (viktarradzin) [reply vradzin/03_HTML/task_2.html]: deleted unnecessary data

**Review by pluhin — APPROVED** [review]
Good , approved

**Pull request merged**